public/video-ui/src/actions/PlutoIdActions/addProject.js

```diff
@@ -0,0 +1,36 @@
+import PlutoVideosApi from '../../services/PlutoVideosApi';
+
+function requestAddProject() {
+  return {
+    type:       'ADD_PROJECT_GET_REQUEST',
```

Comment:
This might be style nitpicking but I don't think these action names convey intent very well.

Maybe instead:

```
ADD_PROJECT_REQUEST
PROJECT_ADDED
```